app/templates/rentals/show.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="jumbo">
```

Comment:
Should we have some kind of back button.  The only way to get back to the rental list atm is to hit the browser back button on click the super rentals logo

![image](https://cloud.githubusercontent.com/assets/3609063/17412110/0a845734-5a49-11e6-8222-bd554d71c80c.png)